src/Compilers/Core/Portable/CodeGen/ILBuilderEmit.cs

```diff
@@ -695,6 +701,24 @@ internal void EmitLongConstant(long value)
             }
         }
 
+        internal void EmitNativeIntConstant(long value)
```

Comment:
>EmitNativeIntConstant [](start = 22, length = 21)

Should/could we emit specific ILOpCodes as we do in `EmitIntConstant` for values between 0 and 8?

Reply:
We do. That is handled in the call to `EmitIntConstant()` below.

---
In reply to: [379037548](https://github.com/dotnet/roslyn/pull/41380#discussion_r379037548) [](ancestors = 379037548)